homeassistant/components/pegel_online/quality_scale.yaml

```diff
@@ -0,0 +1,87 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: no actions/services are implemented
+  appropriate-polling: done
+  brands: done
+  common-modules: done
```

Comment:
`_attr_available = True` in entity.py does nothing.

You could move the device_info property to `_attr_device_info`